### PR TITLE
Added logging and  graceful shutdown

### PR DESCRIPTION
**Pull request by aneeshkp**

### CODE REVIEW

File: events.go [hunk 1]
```diff
@@ -53,22 +56,38 @@ func main() {
...
+	log.Printf("Config %#v\n", serverConfig)
 	eventsNotifier := make(chan string) // Channel for messages from goroutines to main()
 	var amqpEventServer *amqplistener.AMQPServer
 	///Metric Listener
 	amqpEventsurl := fmt.Sprintf("amqp://%s", serverConfig.AMQP1EventURL)
+	log.Printf("Connecting to AMQP1 : %s\n", amqpEventsurl)
 	amqpEventServer = amqplistener.NewAMQPServer(amqpEventsurl, true, -1, eventsNotifier)
+	log.Printf("Listening.....\n")
 	var elasticClient *saelastic.ElasticClient
+	log.Printf("Connecting to Elastic Search : %s\n", serverConfig.ElasticHostURL)
```
Comment (leifmadsen) [inline events.go]: Just to be pedantic, it should be "ElasticSearch"

File: metrics.go [hunk 2]
```diff
@@ -150,13 +152,31 @@ func main() {
 	handler.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
 	handler.HandleFunc("/debug/pprof/trace", pprof.Trace)
 
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt)
+	go func() {
+		for sig := range c {
+			// sig is a ^C, handle it
+			log.Printf("caught sig: %+v", sig)
+			log.Println("Closing http server and listener")
```
Comment (leifmadsen) [inline metrics.go]: You should have the same logs in both applications. This one has an extra line that isn't present in the event consumer.

File: metrics.go [hunk 3]
```diff
@@ -150,13 +152,31 @@ func main() {
...
+	}()
+	log.Printf("Config %#v\n", serverConfig)
 	//run exporter fro prometheus to scrape
 	go func() {
 		metricsURL := fmt.Sprintf("%s:%d", serverConfig.Exporterhost, serverConfig.Exporterport)
+		log.Printf("Metric server at : %s\n", metricsURL)
 		log.Fatal(http.ListenAndServe(metricsURL, handler))
 	}()
+	time.Sleep(2 * time.Second)
+	log.Println("Http server is ready....")
```
Comment (leifmadsen) [inline metrics.go]: HTTP

File: metrics.go (hunk 2 above)
Comment (aneeshkp) [reply metrics.go]: Removed in log.Println("Closing http server and listener") from metrics

File: events.go (hunk 1 above)
Comment (aneeshkp) [reply events.go]: Fixed

**Pull request merged**